### DISCUSSION TITLE
YAML code van screenshot

### DISCUSSION THREAD

**Discussion by SvenKortekaas** [cyberjunky/addon-p2000_rtlsdr#16] [Q&A] [answered]
Ik ben bezig met het maken van een dashboard voor thuis, maar kom er niet helemaal uit. Heb je misschien de YAML code van de screenshot op de readme pagina?

![](https://github.com/cyberjunky/addon-p2000_rtlsdr/blob/main/images/screenshot.png?raw=true)

**Comment by cyberjunky** [comment, marked as answer]
@SvenKortekaas Sorry voor de vertraging, wellicht heb je zelf al eea uitgevonden, dit is de Yaml van een soortgelijk dashboard van mijn development instance ```title: Development
views:
  - path: default_view
    title: Home
    cards:
      - type: vertical-stack
        cards:
          - type: entities
            entities:
              - entity: sensor.p2000_amsterdam
              - entity: sensor.p2000_dordrecht_e_o
              - entity: sensor.p2000_lifeliners
              - entity: sensor.p2000_grip_meldingen
              - entity: sensor.p2000_rijnmond
            title: P2000 Meldingen
      - type: vertical-stack
        cards:
          - type: history-graph
            entities:
              - entity: sensor.p2000_dordrecht_e_o
            title: Events
            hours_to_show: 1
          - type: map
            entities:
              - entity: sensor.p2000_dordrecht_e_o
            default_zoom: 16
            title: Lokatie
          - type: logbook
            entities:
              - sensor.p2000_dordrecht_e_o
            title: Dordrecht
      - type: vertical-stack
        cards:
          - type: logbook
            entities:
              - sensor.p2000_grip_meldingen
            title: GRIP
          - type: logbook
            entities:
              - sensor.p2000_lifeliners
            title: Lifeliners
```

**Reply by SvenKortekaas** [reply]
Dank je wel. Had zelf inderdaad al wat uitgevonden en kom erachter dat Github mij geen mails meer stuurt dus ik heb deze veel te laat gezien.
